profiler/src/profiler.cpp

```diff
@@ -11,6 +11,7 @@ int profile_gemm_bias_2d(int, char*[]);
 int profile_gemm_bias_relu(int, char*[]);
 int profile_gemm_bias_relu_add(int, char*[]);
 int profile_gemm_reduce(int, char*[]);
+int profile_gemm_bias_add_reduce(int, char*[]);
```

Comment:
This might be out of scope of this PR, but I think it makes more sense for profiler to benchmark a complete op. So, `gemm_layernorm` is more appropriate than `gemm_bias_add_reduce`.

With that, we can implement a wrapper device function `device_gemm_layernorm` which calls underlying `device_gemm_bias_add_reduce` and `device_5ary_elementwise`. `device_gemm_layernorm` will share the same interface with fused single kernel gemm_layernorm in #263 with general gemm_layernorm, which allow direct comparison when problem size permits.

@zjing14 @asroy What do you say?

Reply:
device_gemm_layernorm and device_5ary_elementwise will be individually tunable. Putting them into the same device op would make tuning complicated.

On the other hand, if we keep them in separate device op, then clients of CK can individually tune them, which also save a lot of work for us

Reply:
@rocking5566 do you think you would need to tune ```device_5ary_elementwise```? If ```device_5ary_elementwise```doesn't need tuning, maybe @rosenrodt's  idea to put them into same device OP is better?